docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -97,93 +197,175 @@ Windows::
 ----
-<1> If you don't have a Hazelcast Enterprise license key, you can either use a Hazelcast Viridian https://docs.hazelcast.com/cloud/free-trial[trial cluster] or request a xref:getting-started:enterprise-overview.adoc#step-2-get-your-enterprise-license[trial license].
-<2> If you want to use the full distribution, remove the `-slim` postfix. See xref:getting-started:editions.adoc#full-and-slim-distributions[Distributions] to learn about the differences between full and slim distributions.
 
-The cluster tries to start on port `5901` if it's available. It chooses another port otherwise. You can check the port in the cluster logs shown on the terminal.
+. If required, add the `ssl` information. The format is as follows:
++
+----
+ssl:
+      enabled: true
+      properties:
+        keyStore: client.keystore
+        keyStorePassword: abc123
+        trustStore: client.truststore
+        trustStorePassword: abc123
+----
 
-The configuration file for the target cluster is `target.yaml`, which can be found in the folder where you extracted the DMT package.
-In case the cluster starts on a different port as you notice within the cluster logs, you must change the `address` field in `target.yaml` accordingly to match the port number.
+. Save the file
 
-NOTE: Instead of Docker, you can also by https://hazelcast.com/get-started/download/[download] Hazelcast 5.3.2 or newer, request a xref:getting-started:enterprise-overview.adoc#step-2-get-your-enterprise-license[trial license], and xref:getting-started:get-started-binary.adoc#step-2-start-a-local-member[start] a cluster.
+NOTE: For further information on the `ssl` properties and their values, refer to the https://docs.hazelcast.com/cloud/connect-to-cluster#advanced[Using Advanced Setup^] section in the Hazelcast {hazelcast-cloud} documentation.
 
-=== Step 4. Start the Migration Cluster
+=== Start the Migration Cluster
 
-Run the following command in the folder where you extracted the DMT package.
+To start the migration cluster, complete the following steps:
 
+. Open a terminal
+. Navigate to the folder in which you extracted the DMT package
+. Enter the following command:
++
 [source,shell]
 ----
-$ HZ_NETWORK_PORT_PORT=5702 HZ_CLUSTERNAME=migration ./bin/hz start
+HZ_NETWORK_PORT_PORT=5702 HZ_CLUSTERNAME=migration ./bin/hz start
 ----
 
-The cluster tries to start on port `5702` if it's available. It chooses another port otherwise. You can check the port in the cluster logs shown on the terminal.
+NOTE: The DMT will attempt to connect to the migration cluster indefinitely. This means that it can appear to hang if unable to connect. To avoid this, you can set a timeout for the connection attempt using the `--timeout` flag.  For further information on the `--timeout` flag, refer to the https://docs.hazelcast.com/clc/5.3.5/configuration#clc-configuration-with-command-line-parameters[CLC Configuration with Command-line Parameters^] section of the Hazelcast CLC documentation. 
```

Comment:
Thanks for adding this but can you move this right under the "./bin/dmt_[os]_[arch]" command in "run migration"?